ibl_to_nwb/AlyxToNWB/io_tools.py

```diff
@@ -1,24 +1,27 @@
-from lazy_ops import DatasetView
+from datetime import datetime
+from pathlib import PurePath
+
 import numpy as np
 import pandas as pd
-from datetime import datetime
 from tzlocal import get_localzone
-from pathlib import PurePath
 
 
-def iter_datasetview(datasetview_obj):
+def iter_datasetview(datasetview_obj, channel_ids=None):
     """
     Generator to return a row of the array each time it is called.
     This will be wrapped with a DataChunkIterator class.
 
     Parameters
     ----------
-    datasetview_obj: DatasetView
+    datasetview_obj: np.array
         2-D array to iteratively write to nwb.
+    channel_ids: np.array
+        channel numbers to store
     """
-
+    if channel_ids is None:
+        channel_ids = np.array(range(datasetview_obj.shape[1]))
     for i in range(datasetview_obj.shape[0]//700):
-        curr_data = np.squeeze(datasetview_obj[i:i + 1])
+        curr_data = np.squeeze(datasetview_obj._raw[i][channel_ids])
```

Comment:
```suggestion
        curr_data = np.squeeze(datasetview_obj._raw[i, channel_ids])
```

Reply:
this seems to give an error with their dataset. It seems to be stored an array of arrays on their end

Reply:
I'm confused about the form of the dataset. I've never seen this `_raw` thing before. Is this really an np.array as the docstring suggests?